lib/utils.js

```diff
@@ -1,5 +1,26 @@
 let { list } = require('postcss')
 
+class Cache {
+  constructor () {
+    // FIXME: private class field is supported for Node.js 12 and newer
+    this.map = new Map()
```

Comment:
`map` should be a "private" field as it shouldn't be exposed, however `private` is only supported since Node.js 12. A `FIXME` message has been added.